internal/postgresql/postgresql.go

```diff
@@ -877,6 +907,25 @@ func (p *Manager) writePgHba() error {
 		})
 }
 
+func (p *Manager) writePgIdent() error {
+	return common.WriteFileAtomicFunc(filepath.Join(p.dataDir, "pg_ident.conf"), 0600,
+		func(f io.Writer) error {
+			if p.ident != nil && len(p.ident) > 0 {
+				if _, err := f.Write([]byte("# MAPNAME\tSYSTEM-USERNAME\tPG-USERNAME" + "\n")); err != nil {
+					return err
+				}
+				for key, value := range p.ident {
```

Comment:
Entries will likely be written out in an undefined order (https://go.dev/blog/maps#iteration-order) - sorting the entries before writing them out might be beneficial